server/controllers/userController.js

```diff
@@ -28,7 +28,7 @@ const signUp = async (req, res) => {
       const token = generateToken(user._id);
       res.cookie("dolphinToken", token, {
         maxAge: 3600000,
-        sameSite: "none",
+        // sameSite: "none",
```

Comment:
If you're commenting out this code, can it just be deleted?

Reply:
Removed